src/Testing/src/xunit/HelixConstants.cs

```diff
@@ -8,5 +8,5 @@ public static class HelixConstants
     public const string Windows10Arm64 = "Windows.10.Arm64v8.Open;";
     public const string DebianAmd64 = "Debian.11.Amd64.Open;";
     public const string DebianArm64 = "Debian.11.Arm64.Open;";
-    public const string RedhatAmd64 = "Redhat.7.Amd64.Open;";
+    public const string AlmaLinuxAmd64 = "(AlmaLinux.8.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:almalinux-8-helix-amd64;";
```

Comment:
I have no idea whether this is correct, but it _is_ notably different from the others.  In particular, why would it mention both alma and ubuntu?

Reply:
Fancy